### PR TITLE
Updated C28131, C28616

### PR DESCRIPTION
**Pull request by MugBergerFries**
Updated to new format, clarified wording and examples

### CODE REVIEW

File: windows-driver-docs-pr/devtest/28131-driverentry-saving-pointer-to-buffer.md [hunk 1]
```diff
@@ -4,23 +4,49 @@ description: Warning C28131 The DriverEntry routine should save a copy of the ar
...
+```cpp
+UNICODE_STRING g_RP;
 
+NTSTATUS
+DriverEntry(
+    PDRIVER_OBJECT DriverObject,
+    PUNICODE_STRING RegistryPath
+    )
+{
+    g_RP = CloneRgistryPath(RegistryPath);  // CloneRegistryPath is an example helper function that copies over the data.
```
Comment (Rastaban) [inline windows-driver-docs-pr/devtest/28131-driverentry-saving-pointer-to-buffer.md:49]: Spelling: CloneRegistryPath

**Review by Rastaban — APPROVED** [review]

**Review by DOMARS — APPROVED** [review]

File: windows-driver-docs-pr/devtest/28131-driverentry-saving-pointer-to-buffer.md [hunk 2]
```diff
@@ -4,23 +4,49 @@ description: Warning C28131 The DriverEntry routine should save a copy of the ar
 keywords:
 - warnings listed WDK PREfast for Drivers
 - errors listed WDK PREfast for Drivers
-ms.date: 04/20/2017
-f1_keywords: 
-  - "C28131"
+ms.date: 09/27/2022
+f1_keywords: ["C28131", "NOT_COPYING_NAME", "NOT_COPYING_NAME"]
```
Comment (corob-msft) [inline windows-driver-docs-pr/devtest/28131-driverentry-saving-pointer-to-buffer.md:8]: "NOT_COPYING_NAME" appears twice in the f1_keywords. Is one of the values missing something? `__WARNING_` perhaps?

File: windows-driver-docs-pr/devtest/28616-multithreaded-av-condition.md [hunk 3]
```diff
@@ -1,53 +1,57 @@
...
 
-Another thread T2 executes lines 1, 2, and 3 decrements **m\_cRef** to 0. Then it executes lines 4 and 5, where **this** is deleted, and finally executes line 6.
+A thread T1 executes the `if` condition, decrements `m_cRef` to 1, and is then preempted.
 
-When T1 is rescheduled, it will reference **m\_cref** on line 9. Thus it will access a member variable after the related this pointer has been deleted—and when the heap for the object is in an unknown state.
+Another thread T2 executes the `if` condition, decrements `m_cRef` to 0, executes the `if` body (where `this` is deleted), and returns `NULL`.
+
+When T1 is rescheduled, it will reference `m_cref` on line 9. Thus it will access a member variable after the related `this` pointer has been deleted—and when the heap for the object is in an unknown state.
 
 ```
```
Comment (corob-msft) [inline windows-driver-docs-pr/devtest/28616-multithreaded-av-condition.md:29]: Add a `cpp` to these example code blocks so we get code colorization.

File: windows-driver-docs-pr/devtest/28131-driverentry-saving-pointer-to-buffer.md (hunk 1 above)
Comment (MugBergerFries) [reply windows-driver-docs-pr/devtest/28131-driverentry-saving-pointer-to-buffer.md]: Done

File: windows-driver-docs-pr/devtest/28131-driverentry-saving-pointer-to-buffer.md (hunk 2 above)
Comment (MugBergerFries) [reply windows-driver-docs-pr/devtest/28131-driverentry-saving-pointer-to-buffer.md]: Done

File: windows-driver-docs-pr/devtest/28616-multithreaded-av-condition.md (hunk 3 above)
Comment (MugBergerFries) [reply windows-driver-docs-pr/devtest/28616-multithreaded-av-condition.md]: Done

**Review by MichaelSquires — APPROVED** [review]

**Pull request merged**